opennms-doc/guide-install/src/asciidoc/text/basic-setup-opennms/rhel/step-3-initialize-opennms.adoc

```diff
@@ -24,3 +24,14 @@ systemctl enable opennms
 ----
 systemctl start opennms
 ----
+
+.Allow connection to the Web UI from your network
+[source, shell]
+----
+firewall-cmd --permanent --add-port=8980/tcp
+systemctl reload firewalld
+----
+
+TIP: If you want to receive SNMP Traps or Syslog messages you have to allow incoming traffic on your host firewall as well.
+     By default OpenNMS SNMP trap daemon is listening on 162/udp and Syslog daemon is listening on 10514/udp.
```

Comment:
Out of scope for this PR, but we should probably add some firewalld `services` configs for OpenNMS so we could make nicer versions of this.